app/Http/Requests/BlockImportRequest.php

```diff
@@ -25,6 +25,11 @@ public function rules() {
      * @return BlockListImporter
      */
     public function getImporter() {
-        return app()->get(ImportServiceProvider::$BLOCK_IMPORTER_MAP[$this->input('source')]);
+        // check if it is a pdf from ecamp3
```

Comment:
Eigentlich war mal genau der hidden Input namens "source" dafür angedacht, die Herkunft der Datei zu entscheiden. Also die Idee wäre, für den Import aus eCamp v3 ein separates Formular mit Anleitung, eingeschränkten Dateitypen im File Inout Element und mit source=eCamp3BlockList zu erstellen.

Reply:
Jap das macht sicher Sinn, habe mich gefragt, ob man es separat oder zusammen machen soll. Aktuell schaut es auf einen pdf mimeType für ecamp3 und alles in einem Form. Man könnte auch ein Dropdown machen wo man die source auswählt?
Aber ich mache sonst mal zwei separate imports.